.github/workflows/builder.yaml

```diff
@@ -12,6 +12,10 @@ on:
     branches:
       - main
 
+permissions:
+  contents: read
+  packages: write
```

Comment:
This is not what the title of the PR said, maybe make this a separate PR? Additionally, it might be better to only pass write permissions to the specific tasks that need those (instead of the whole job).

Reply:
Apologies, it snuck in with the other changes - I have removed it